src/sonic-host-services/scripts/hostcfgd

```diff
@@ -240,6 +449,37 @@ class AaaCfg(object):
         with open(NSS_TACPLUS_CONF, 'w') as f:
             f.write(nss_tacplus_conf)
 
+        # Set debug in nss-radius conf
+        template_file = os.path.abspath(NSS_RADIUS_CONF_TEMPLATE)
+        template = env.get_template(template_file)
+        nss_radius_conf = template.render(debug=self.debug, trace=self.trace, servers=radsrvs_conf)
+        with open(NSS_RADIUS_CONF, 'w') as f:
+            f.write(nss_radius_conf)
+
+        # Create the per server pam_radius_auth.conf
+        if radsrvs_conf:
+            for srv in radsrvs_conf:
+                # Configuration File
+                pam_radius_auth_file = RADIUS_PAM_AUTH_CONF_DIR + srv['ip'] + ":" + srv['auth_port'] + ".conf"
+                template_file = os.path.abspath(PAM_RADIUS_AUTH_CONF_TEMPLATE)
+                template = env.get_template(template_file)
+                pam_radius_auth_conf = template.render(server=srv)
+
+                open(pam_radius_auth_file, 'a').close()
+                os.chmod(pam_radius_auth_file, 0o600)
+                with open(pam_radius_auth_file, 'w+') as f:
+                    f.write(pam_radius_auth_conf)
+
+        # Start the statistics service
+        cmd = 'service aaastatsd start'
```

Comment:
Do we stop this, when remote login is *not* enabled ?

Or may be OK to run it always, irrespective of AAA is active or not ?